### PR TITLE
Add force delete for hotels

### LINKED ISSUE #36: add 'force remove hotel' endpoint
hotel owner should be able to remove hotel with resrvations as well. 
You can modify current endpoint and add query param to the request endpoint

remove all reservations when flag is set to true

change message to:
    if (reservation.length > 0)
      return res.status(400).send('Remove reservations first, please')

when hotel was deleted successfuly send all hotels for this owner

### PR DESCRIPTION
**Pull request by GoraJakub**

### CODE REVIEW

**Review by bartoszbialecki — COMMENTED** [review]
Did you not have this code in other PR?

File: config/default.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
-  "jwtPrivateKey": "",
+  "jwtPrivateKey": "12345",
```
Comment (pdybowski) [inline config/default.json]: Remove its value

File: routes/owner.js [hunk 2]
```diff
@@ -2,62 +2,78 @@ const mongoose = require('mongoose')
...
   const { error } = validate(req.body)
-  if (error) return res.status(400).send(error.details[0].message)
+  if (error) throw new ApiError(400, error.details[0].message)
 
   const hotel = new Hotel(req.body)
 
   await hotel.save()
 
-  res.status(200).send(hotel)
+  res.status(200).json({ hotel: hotel })
```
Comment (pdybowski) [inline routes/owner.js]: On front end:
Const hotel = res.hotels
Why we can't make it as simple as possible...
Just send(hotel) lol

File: routes/hotels.js [hunk 3]
```diff
@@ -1,37 +1,38 @@
...
 const express = require('express')
 const { Hotel } = require('../models/hotel')
+const ApiError = require('../helpers/apiError')
 const router = express.Router()
 
 router.get('/', async (req, res) => {
   const hotels = await Hotel.find()
 
-  res.status(200).send(hotels)
+  res.status(200).json({ hotels: hotels })
```
Comment (pdybowski) [inline routes/hotels.js]: Why.. you are making front devs job more difficult.
Send(hotels) 

Sending objects makes sense in error 
Const error = err.message 
Where we want to exclude message

File: routes/owner.js [hunk 4]
```diff
@@ -2,62 +2,78 @@ const mongoose = require('mongoose')
...
+        400,
+        'Remove reservations first or set flag force to true, please'
+      )
 
     await Hotel.findByIdAndDelete(id)
-    res.status(200).send('Hotel deleted')
+
+    const hotels = await Hotel.find()
+    res.status(200).json({
+      message: `Hotel ${message}deleted`,
```
Comment (pdybowski) [inline routes/owner.js]: We don't want this message. We can make it on front

**Review by pdybowski — COMMENTED** [review]
You are making your job more difficult

File: routes/hotels.js (hunk 3 above)
Comment (GoraJakub) [reply routes/hotels.js]: I was told to make everything json, but no problem 😄

**Pull request merged**